docs/notebooks/03_merge.ipynb

```diff
@@ -1,7 +1,6 @@
 {
```

Comment:
<p>you can use</p><pre class="ql-syntax" spellcheck="false">:::{note}
asfasdasdfasdf
:::
</pre><p> </p><p>and it will render as a nice box in sphinx. There are also other types of "admonitions", see the [myst docs](https://myst-parser.readthedocs.io/en/latest/syntax/admonitions.html)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/icbi-lab/atlas_protocol/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>actually you can see a dramatic difference in matched ids if using the reference that was originally used or the latest one!</p><p>Don't know if it might be interesting to point this out, because it shows how unreliable gene symbols can be matched across different annotations. In fact I would argue that this is the main reason to go through the extra effort to find the used reference, download it, etc. instead of updating the gene symbols directly.</p>

Reply:
<p>I think it's quite an elegant solution and probably worth the effort. </p>